### PR TITLE
NMS-14979: Fix service naming

### PR DESCRIPTION
**Pull request by fooker**
https://issues.opennms.org/browse/NMS-14979

### CODE REVIEW

File: plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
 package org.opennms.velocloud.requisition;
 
+import java.net.Inet4Address;
```
Comment (aramosvizcarra) [inline plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java:31]: I think this 2 are not being used.

Comment (aramosvizcarra) [inline plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java:31]:
```suggestion

```

File: plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java [hunk 2]
```diff
@@ -28,6 +28,8 @@
...
+import java.net.InetAddress;
```
Comment (aramosvizcarra) [inline plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java:32]:
```suggestion

```

**Review by aramosvizcarra — CHANGES_REQUESTED** [review]
just a minor change

File: plugin/src/main/resources/poller-configuration.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <poller-configuration xmlns="http://xmlns.opennms.org/xsd/config/poller/api">
     <package name="velocloud">
-        <filter>IPADDR != '0.0.0.0'</filter>
+        <filter>IPADDR != '255.255.255.255'</filter>u
```
Comment (christianpape) [inline plugin/src/main/resources/poller-configuration.xml:3]: I really do not like this. Maybe we should pick the first interface or the first public IP or something like that.

**Review by christianpape — APPROVED** [review]
LGTM.

File: plugin/src/main/resources/poller-configuration.xml (hunk 3 above)
Comment (fooker) [reply plugin/src/main/resources/poller-configuration.xml:3]: After some discussion, we figured that this is the best way to do this.

**Review by christianpape — APPROVED** [review]
LGTM.

**Pull request merged**